hop/models.py

```diff
@@ -82,7 +86,9 @@ def asdict(self):
         return asdict(self)
 
     def __str__(self):
-        headers = [(name.upper() + ":").ljust(9) + val for name, val in self.header.items()]
+        headers = [
```

Comment:
This is out of scope for this PR, but I would be inclined to pick up the `autopep8` with associated config as @spenczar added for `adc-streaming` as a future task, as we can allow the 100 line limit in our 'format guide' and avoid formatting stuff like this.